tests/sequence_rooter.rs

```diff
@@ -0,0 +1,74 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#![cfg(feature = "debugmozjs")]
```

Comment:
Let's remove this so the test is always run.

Reply:
Ah, `JS_SetGCZeal` needed that and I thought I stumbled upon this also with `JS_GC`, but that's not the case and it's not needed here. Will do, thanks!